tests/phpunit/tests/media.php

```diff
@@ -3558,6 +3578,19 @@ function() {
 		$content_expected   = $img1 . $iframe1 . $lazy_img2 . $lazy_img3 . $lazy_iframe2;
 		$content_expected   = wp_img_tag_add_decoding_attr( $content_expected, 'the_content' );
 
+		// add dominant color markup
```

Comment:
What is happening here?

Reply:
Now that dom color is behind a theme flag we need to check that path

Reply:
Sorry left behind removed